meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_%.bbappend

```diff
@@ -1,3 +1,11 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: MIT
+# ---------------------------------------------------------------------------------------------------------------------
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/lttng-tools:"
```

Comment:
should we be using the staging featureoverride here? and in the other places?

Reply:
Yes, thanks!